ci-operator/config/openshift/assisted-test-infra/openshift-assisted-test-infra-master.yaml

```diff
@@ -233,6 +233,14 @@ tests:
         OPENSHIFT_VERSION=4.14
       TEST_SUITE: full
     workflow: assisted-baremetal
+- as: e2e-metal-assisted-4-15
+  optional: true
```

Comment:
```suggestion
  optional: true
  always_run: false
```

Reply:
I don't see any `always_run: false` in the rest of the tests but they don't run automatically no?

Reply:
> before we were updating the job directly
here: https://github.com/openshift/release/pull/45375/files#diff-e52081e3ba3e344ffd4d8ffb587d28fcb4366b5207c692138f512a11b8f7d404R484

Reply:
Oh because it is new. got it